ideascube/management/commands/tags.py

```diff
@@ -121,3 +135,53 @@ def list(self, options):
         print(row.format('.' * 40, '.' * 40, '.' * 40))
         for tag in Tag.objects.order_by('slug'):
             print(row.format(tag.name, tag.slug, self._count(tag.name)))
+
+    def sanitize(self, options):
+        all_tags = list(Tag.objects.all().order_by('id').reverse())
```

Comment:
Hum, it was when we wanted to keep the case of the first created tag.
But this is not relevant now.